test/mocks/PriceSourceMock.sol

```diff
@@ -19,4 +19,14 @@ contract PriceSourceMock {
         decimals = _decimals;
     }
 
-}
\ No newline at end of file
+    function latestRoundData() external view returns (
+        uint80 roundId,
+        int256 answer,
+        uint256 startedAt,
+        uint256 updatedAt,
+        uint80 answeredInRound
+    ) {
+        return (0, latestAnswer, 0, 0, 0);
```

Comment:
This should be returning all values from the tuple since they're all returned from the oracle, can add another setter setLatestRoundData to set these values within the mock.

Reply:
I've further restricted what gets returned and refactored the interface setup to not overload the price source one.